diff --git a/node_modules/investing-economic-calendar/dist/crawler.js b/node_modules/investing-economic-calendar/dist/crawler.js
index 26a7478..0a4441c 100644
--- a/node_modules/investing-economic-calendar/dist/crawler.js
+++ b/node_modules/investing-economic-calendar/dist/crawler.js
@@ -35547,8 +35547,4 @@ async function fetchEconomicEvents(params) {
   });
   return events;
 }
-export {
-  fetchEconomicEvents
-};
-
 export { Country, Currency, Importance, fetchEconomicEvents };
